libtiledbvcf/src/read/reader.cc

```diff
@@ -779,7 +779,7 @@ std::pair<size_t, size_t> Reader::get_intersecting_regions_v3(
   size_t original_last = last;
 
   // Next find the index of the last region that intersects the cell's
-  // REAL_START position. This is used as the actual interval of intersection.
+  // REAL_START position. This is used as the actual interval of intersection
```

Comment:
Accidental? We should probably keep the period.

Reply:
Yeap, must have fat-fingered the delete key